stellarphot/settings/models.py

```diff
@@ -294,15 +294,14 @@ def validate_gain(self):
         # Check that gain and read noise have compatible units, that is that
         # gain is read noise per data unit.
         gain = self.gain
-        if (
-            len(gain.unit.bases) != 2
-            or gain.unit.bases[0] != rn_unit
-            or gain.unit.bases[1] != self.data_unit
-        ):
+
```

Comment:
I have a vague memory that this may be my code you are fixing... good, it looks ugly now after seeing so much pydantic in your commits over the last few months. :). Do we want to change the dark current check a few lines below this to take a similar approach?

Reply:
> Do we want to change the dark current check a few lines below this to take a similar approach?

Maybe...opening a separate issue for that